sickchill/views/music.py

```
@@ -0,0 +1,338 @@
+import logging
+import os
+
+from sickchill import settings
+from sickchill.oldbeard import config
```

Ruff (0.12.2)

`sickchill.oldbeard.config` imported but unused

Remove unused import: `sickchill.oldbeard.config`

(F401)